### PR TITLE
Added readme file

### PR DESCRIPTION
**Pull request by beccamason**

### CODE REVIEW

**Review by nmunro — CHANGES_REQUESTED** [review]
Always good to have a README but can you add a little more detail?

File: README.md [hunk 1]
```diff
@@ -1 +1,3 @@
-# Artist Database 
\ No newline at end of file
+# Artist Database!
+
+Add your fave painters to your list!
```
Comment (nmunro) [inline README.md:3]: Perfect, thanks for this!

**Review by nmunro — APPROVED** [review]
Great job!

**Review by dannyvsm — COMMENTED** [review]
LGTM

**Pull request merged**